packages/component-base/test/url-utils.test.js

```diff
@@ -29,5 +30,10 @@ describe('url-utils', () => {
         expect(arePathsMatching(pathWithLeadingSlash, pathWithoutLeadingSlash)).to.be.true;
       });
     });
+
+    it('should ignore base uri in paths', () => {
+      sinon.stub(document, 'baseURI').value('https://base');
```

Comment:
I'm afraid stubs aren't restored automatically, so it can potentially affect the future following tests.